.github/workflows/rebase-suite-e.yml

```diff
@@ -0,0 +1,45 @@
+name: Rebase the suite-e branch on develop
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  rebase:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Fetch the suite-e branch
+        run: git fetch origin suite-e:suite-e
+      - name: Rebase the suite-e branch on develop
+        id: rebase
+        run: |
+          git checkout suite-e
+          echo 'Rebasing suite-e to develop'
+          git rebase develop
```

Comment:
see the comments above.

Reply:
f43c2e4ac4

Reply:
there is still git checkout suite-e which is not needed now as the action checkout does that